### DISCUSSION TITLE
EarnApp Warning

### DISCUSSION THREAD

**Discussion by superit63** [engageub/InternetIncome#126] [General]
Hello all,
After running Earnapp with residential proxy, my account has been suspended. Here is the screenshot:
![image](https://github.com/engageub/InternetIncome/assets/169600953/88ee9565-0cdb-4321-a264-2609b7b0f57e)
I think they can detect we are using docker because they cant detect the proxy (residential proxy is the best in class)
Everyone should be carefully when running Earnapp or you can choose others site.
Thanks

**Comment by engageub** [comment]
Hello,
It may be due to your rotating proxy that you used today.

Thank you

**Comment by superit63** [comment]
Nope. I have tested 2 accounts
1 account with static residential proxy (suspended)
1 account with rotating proxy (not suspended maybe not yet payout)

Tran Quang Sieu

**Reply by engageub** [reply]
Was it showing green colour in dashboard. How many nodes were you running.

Thank you

**Reply by engageub** [reply]
Would you be able to find the difference between bare metal server and cloud vps?
OS is installed and resources are allocated. 
Virtual Machines, LXD and Docker containers, do similar job of isolating the environment.
You have to make sure you are using high quality Residential IP. 
Its totally upto you, to use direct connection or proxies. 
The script has an option to support multi proxies which you can use it accordingly. 

Thank you

**Comment by superit63** [comment]
Yes it was green in dashboard, definitely. I was running 20 nodes (before I
run 40 to test them I decreased to 20 nodes). How many nodes will be ok?
Btw, is there any possibility to detect we are using docket or not? And as
far as I know, honeygain can detect the docker and they allow to use
docker. I dont know why Earnapp doesnt allow it

Tran Quang Sieu

**Reply by engageub** [reply]
I think the number of nodes was the problem. Earlier the dashboard limited to 15 nodes and the table was updated to 15 earnapp nodes per account. Later it was set to unlimited as the FAQ did not mention it. 

Thank you

**Comment by superit63** [comment]
Ok, thanks for the clarification.

I will look into it and use around 15 nodes/account. I will update if I get
banned.

**Reply by engageub** [reply]
I just updated the Readme table to use only 10 nodes per account, although the user can still add nodes.

Thank you

**Comment by superit63** [comment]
Yeppp. No way to find the difference if they are isolated environments.
Your code will do it. Right?
I can be sure that my IP is good enough, the quality score is good when i
checked on ipqualityscore

Tran Quang Sieu

**Comment by engageub** [comment]
Thank you, please make sure you are using only static IP not rotating proxy.
You have the option to use direct connection or proxies.
I do not own any of the docker images. Either they are official or published by users who have already tested it.
The script is an integration of multiple docker images which has the ability to support multi proxy,ip and VPN.
It is upto the users to decide on what apps to use proxies for.


Thank you

**Comment by engageub** [comment]
Please do not leave the node IDs idle in the dashboard. There are chances that same node ID can be assigned to  other user.
I have come across this earlier, when my nodes were offline and some other user had used the same node id and it was showing online on my dashboard.
Make sure your node id is online if it is present in dashboard and do not leave it idle for long time.

Thank you

**Comment by superit63** [comment]
Thanks a lot Engageub.
Here is the summary for Earnapp that will be useful for new user like me:
1. Dont try to add many nodes in 1 account. Maximum is 10 nodes/account
2. Dont use rotating proxy, only static proxy
3. Dont let node idle in dashboard, delete the idle node
